boa/test/plugin.py

```diff
@@ -1,15 +1,27 @@
+import contextlib
 from typing import Generator
 
 import hypothesis
 import pytest
 
 import boa
+from boa.profiling import get_call_profile_table, get_line_profile_table
+from boa.vm.gas_meters import ProfilingGasMeter
 
 # monkey patch HypothesisHandle. this fixes underlying isolation for
 # hypothesis.given() and also hypothesis stateful functionality.
 _old_init = hypothesis.core.HypothesisHandle.__init__
 
 
+@contextlib.contextmanager
+def _toggle_profiling(yes: bool = False):
```

Comment:
this function is confusing. if the flag is False, it doesn't disable profiling. also, it's only used in one place - maybe just inline it there?